src/app/@ansyn/ansyn/modules/overlays/reducers/overlays.reducer.ts

```diff
@@ -70,6 +70,7 @@ export interface IOverlaysState extends EntityState<IOverlay> {
 	overlaysCriteria: IOverlaysCriteria;
 	miscOverlays: IOverlaysHash;
 	customOverviewElement: any;
+	totalOverlays: IOverlay[];
```

Comment:
if we only need this for the length of the array,
maybe is better to save the length directly.